validator/client/src/main/java/tech/pegasys/teku/validator/client/SyncCommitteeScheduler.java

```diff
@@ -100,6 +107,13 @@ public void onSlot(final UInt64 slot) {
     }
   }
 
+  private boolean isFirstSlotOfEpoch(final UInt64 slot) {
+    final UInt64 currentEpoch = spec.atSlot(slot).miscHelpers().computeEpochAtSlot(slot);
+    final UInt64 startSlotAtEpoch =
+        spec.atEpoch(slot).miscHelpers().computeStartSlotAtEpoch(currentEpoch);
```

Comment:
below we used `spec.computeStartSlotAtEpoch` which was a nice shortcut...

could be something like `spec.computeStartSlotAtEpoch(spec.computeEpochAtSlot(slot)).equals(slot)`

Reply:
may be cleaner to leave currentEpoch defined and just cleaner :)
```
final UInt64 currentEpoch = spec.computeEpochAtSlot(slot);
final UInt64 startSlotAtEpoch = spec.computeStartSlotAtEpoch(currentEpoch);
return startSlotAtEpoch.equals(slot);
```